Sources/arm/ui/UIMenu.hx

```diff
@@ -158,24 +158,18 @@ class UIMenu {
 					menuFill(ui);
 					var light = iron.Scene.active.lights[0];
 					var lahandle = Id.handle();
-					if (lahandle.value < 0) {
-						lahandle.value += (Std.int(-lahandle.value / 360) + 1) * 360;
-					}
-					else if (lahandle.value > 360) {
-						lahandle.value -= Std.int(lahandle.value / 360) * 360;
-					}
+					lahandle.value = Context.lightAngle / Math.PI * 180;
 					menuAlign(ui);
-					var lightAngle = lahandle.value;
-					ui.slider(lahandle, tr("Light Angle"), 0.0, 360.0, true, 1);
-					var ldiff = lahandle.value - lightAngle;
-					if (ldiff != 0) {
-						ldiff = (ldiff) / 180.0 * Math.PI;
+					var newAngle = ui.slider(lahandle, tr("Light Angle"), 0.0, 359.0, true, 1) / 180 * Math.PI;
```

Comment:
Any issues when angle is set to 360?

Reply:
I thought it is stupied to have one angle twice. If the user sets the slider to 360 degrees it automatically jumps to 0 = 2Pi modulo 2Pi

Reply:
Aha, I see. In theory the user can enter any value manually, not sure how it was done before but probably better to handle that case as well.

Reply:
Previously we had
```
else if (lahandle.value > 360) {
lahandle.value -= Std.int(lahandle.value / 360) * 360;
}
```
Of course we could do that, too but the currently implementation assumes we store normalized radians i.e. [0,2pi) (half open interval)

Reply:
Oki, might be good enough to handle it that way for now, since envmap / brush angles also go up to 360 I think.